packages/babel-polyfill/src/index.js

```diff
@@ -1,7 +1,13 @@
+import "core-js/shim";
+import "regenerator-runtime/runtime";
+
 if (global._babelPolyfill) {
-  throw new Error("only one instance of @babel/polyfill is allowed");
+  console.warn(
+    "@babel/polyfill is loaded more than once on this page. This is probably not desirable/intended " +
+      "and may have consequences if different versions of the polyfills are applied sequentially. " +
+      "If you do need to load the polyfill more than once, use @babel/polyfill/noConflict " +
```

Comment:
I think this has to be `@babel/polyfill/lib/noConflict` right? since it's not the main file.